contracts/test/TestJBveBanny.t.sol

```diff
@@ -60,55 +68,131 @@ contract JBveBannyTests is TestBaseWorkflow {
   }
 
   function mintIJBTokens() public returns (IJBToken) {
-    IJBToken _token = _jbTokenStore.tokenOf(_projectId);
+    IJBToken _jbToken = _jbTokenStore.tokenOf(_projectId);
     _projectOwner = projectOwner();
     evm.startPrank(_projectOwner);
     _jbController.mintTokensOf(_projectId, 100 ether, _projectOwner, 'Test Memo', true, true);
-    _token.approve(_projectId, address(_jbveBanny), 10 ether);
-    return _token;
+    _jbToken.approve(_projectId, address(_jbveBanny), 10 ether);
```

Comment:
this shouldn't be part of a minting helper -> suggest moving it to the test itself

Reply:
approve is common in all methods so let me change the method name instead